### DISCUSSION TITLE
Turning "Public goods" into "Club goods". The libertarian project of decentralized Blockchain and how it would destroy the fabric of society

### DISCUSSION THREAD

**Discussion by Walther-Schiller** [rufuspollock/awesome-crypto-critique#188] [Ideas]
Turning Public goods into Club Goods . 
Part 1: Introduction 

A public good is a good with two characteristics : 
1-Marginal cost = 0: (it costs to produce it, but not to create an additional copy. Only a fix cost, no variable cost) 
2- Non excludable : once it is there , you have no mean to make people paying to use it . You cannot exclude people who are not paying. 

As there is no market for a public good, it is what we often call a common . 

A Club good has marginal cost=0 , but which is excludable. You can see here that a club good can be a forme public good that we managed to make excludable . 

The beginning of internet and its culture was to give for free immaterial goods (culture , knowledge) . Problem is that by stripping them from their revenue (Copyright enfrigment, piracy) , you push those "club goods" into the field of the commons . You destroy the market for it . It was a collective libertarianism (you can see that word communism comes from commons) 

The new internet , right wing libertarians, is actually trying to do the opposite. Using technology to make a market of what was commons .  They will argue with the theory of "the tragedy of the commons". And they will also argue about  property as being one of the most important human rights. 

You can see here that thinking this complex issue on society, human rights and the role of state or institions, instead of a nuanced conversation, we can very easaly go down the rabitt hole of two confronting radical ideologies. 

There is a link between the radical libertarian view and the fact that "apes" are often acting like hyper individualistic , narcissic , antisocial , dickheads. The whole project of decentralized blockchains, with the micropayments, the immutable property rights, the anti institution mantra etc is to turn everything into a market , to destroy all the public goods to privatize and monetatized every aspect of our lives . This is very well explained in the now famous youtube video "Lines goes up: the problem with NFT's" https://www.youtube.com/watch?v=YQ_xWvX1n9g

I stop here for now.

**Comment by rufuspollock** [comment]
I think the basic thrust has a lot of merit to it and would be worth writing up (perhaps just write in a gentle way 😉).

Imagine could break into chunks and do one at a time:

1. Good introduction to public goods, private goods, club goods would be *really* useful. Note we already have the start of a page on public goods here https://web3.lifeitself.us/concepts/public-goods
2. Explanation of how the crypto projects is turning digital public goods into club goods with concrete examples. Make effort to steel man the overall argument for why tod do that and then critique.

**Reply by Walther-Schiller** [reply]
Thank you for the advices. I saw the page on public goods just after I posted this one (I am really new on this thing which is Github. I do not know how to structure my posts ). 

I was planning to write every new chapter on each example ( Cuban on the "cryptocritic corner" with his project of making nft of text books chocked me so much) . Which would allow me to explain a bit deeper how you make a public good excludable (transforming it into a club good)  using new technologies and how this is allowing the libertarian fantasy to have finally tools to make real what was impossible. And that would lead me on how making something possible does not mean making it desirable, because economics show us that Markets are not always efficient , and because some goods have huge externalities . This a bit boring for me because it is something I wrote already and was a researcher on . 

But the interesting thing for me (and hopefully for the reader) is how the extralities of crypto can be a whole new breed, because of the pure greed , e.g.  : burning a banksy to give "value" to an NFT of this banksy  https://www.orartswatch.org/the-value-of-art-cats-cryptoart-and-morons/

I will keep in mind to be gentle, but when people are planning to burn art pieces or to extract as much value as possible from textbooks, I have to admit it is not an easy task  ;)  .

**Reply by rufuspollock** [reply]
And like one example per chapter approach. Let's start with just one example ...

**Reply by Walther-Schiller** [reply]
I will.
Mother is in hospital. Have to travel asap. Big stress .
